sdl-snowflake/src/test/scala/io/smartdatalake/workflow/snowflake/SparkAndSnowparkDataPipelineIT.scala

```diff
@@ -61,15 +63,16 @@ object SparkAndSnowparkDataPipelineIT extends App {
   val srcDO = HiveTableDataObject( "src1", Some(tempPath+s"/${srcTable.fullName}"), table = srcTable, numInitialHdfsPartitions = 1)
   srcDO.dropTable
   instanceRegistry.register(srcDO)
-  val tgt1Table = Table(Some("test"), "tgt1", None, Some(Seq("lastname")))
-  val tgt1DO = SnowflakeTableDataObject( "tgt1", tgt1Table, connectionId = "sfCon")
+  val tgt1Table = Table(Some(System.getenv("SNOWFLAKE_SCHEMA")), "tgt1", None, Some(Seq("lastname")))
```

Comment:
@zzeekk  FYI I changed your test to use environment variables. I hope it makes it a bit easier to see how to run the test (no confusion with db vs schema)

Reply:
ok